docs/Using-Fleet/Fleet-UI.md

```diff
@@ -87,13 +87,15 @@ How to update agent options:
 
 2. On the Organization settings page, select **Agent options** on the left side of the page.
 
-3. To see all agent options, head to the [agent options documentation](https://fleetdm.com/docs/deploying/configuration#agent-options).
```

Comment:
#agent-options section does not exist in this document. Updated link to configuration-files#agent-options which is the appropriate section